src/app/[locale]/incident/components/IncidentFormFooter.tsx

```diff
@@ -56,12 +61,16 @@ const IncidentFormFooter = ({
           </Button>
         )}
         {step === 4 && (
+          // Note: current button has no visual indicator when disabled.
           <Button
             variant="primary"
             type="submit"
-            className="justify-self-end"
+            disabled={loading}
```

Comment:
```suggestion
            disabled={loading}
            tabIndex={loading ? 0 : undefined}
```

Reply:
Standaard is een disabled button niet focusable. Om toegang te geven tot de aria-describedby moet de button wel focusable zijn.

Om te voorkomen dat een toekomstige developer denkt: buttons zijn standaard al focusable, ik ruim deze code mooi op, stel ik voor de `tabindex="0"` te koppelen aan de disabled button.

Reply:
Goeie suggestie 👍